### ISSUE TITLE
improved "rename and move, cover, etc" action

### ISSUE THREAD

**Issue by slippycheeze** [seanap/Plex-Audiobook-Guide#30]
I edited your action scripts to unify the "single file" and "multiple file" cases using the script logic.  This works well for me, and I think I covered most of the edge cases.  You might want to incorporate this, or something similar, into your setup.

Inline, not attached, because of github attachment file type issues.

```
[#0]
T=5
1=X:\\audiobooks\\%albumartist%\\%series%\\%album% (%year%)[ '['%series% %series-part%']']\\%album% (%year%)[ '['%series% %series-part%']']$ifgreater(%_total%,1,- Part $num(%track%,2),)
F=_FILENAME

[#1]
T=13
1=%album% (%year%) ['['%series% %series-part%']' ]- cover
3=0|3
2=0

[#2]
T=15
F=desc
1=X:\\audiobooks\\%albumartist%\\%series%\\%album% (%year%)[ '['%series% %series-part%']']\\desc.txt
2=0
3=0

[#3]
T=15
F=reader
1=X:\\audiobooks\\%albumartist%\\%series%\\%album% (%year%)[ '['%series% %series-part%']']\\reader.txt
2=0
3=0
```

**Comment by seanap** [comment]
Awesome! Thanks for sharing

**Comment by seanap** [comment]
Finally had some time to test this out and the `$ifgreater` works great! I will definitely be simplifying the guide to use this approach.

I did notice that for some reason `%_total%` was not working, when looking at my extended tags there is one called `totaltracks` so I used that instead and now it's working.

I also added another rule; If single track set the Title to the new filename.  If multitrack do nothing.  This way if you have chapter names it doesn't overwrite them, and for a single book it cleans up the main book title while leaving the embedded chapter titles alone.

I modified the folder name back to `.../year - title [series]/...` this keeps it consistent with previous versions of the guide  and also increases compatibility with other server apps like Booksonic and Audiobookshelf.
